Makefile

```diff
@@ -25,3 +25,7 @@ pygments:
 
 build:
 	pip install -Ue .[docs,test]
+
+
+clean:
+	git clean -Xn -e "!/*.code-workspace" -e "!/*.vscode"
```

Comment:
Hm, can you add these to `.gitignore`?

Reply:
Sure, will do.

Just note that this command says: "don't remove those". I have them in my global gitignore in my dotfiles and `git clean -X` also looks at your ~/.gitignore file. I don't *want* it to zap my workspace file, all of my configuration is there.

Reply:
Most of us use vscode so having these in .gitignore makes sense.